pkg/loop/internal/pb/reporting_plugin_service.proto

```diff
@@ -0,0 +1,28 @@
+syntax = "proto3";
+
+option go_package = "github.com/smartcontractkit/chainlink-relay/pkg/loop/internal/pb";
+
+package loop;
+
+service ReportingPluginService {
+  rpc NewReportingPluginFactory (NewReportingPluginFactoryRequest) returns (NewReportingPluginFactoryReply) {}
+}
+
+message ReportingPluginServiceConfig {
+  string providerType = 1;
+  string command = 2;
+  string pluginName = 3;
+  bytes pluginConfig = 4;
```

Comment:
So we use `[]byte` in go because that is standard for encoded JSON, but I wonder if we would prefer `string` here? If there are no other tradeoffs, then `string` seems slightly more debuggable for a human.